kubernetes/windows/setup.ps1

```diff
@@ -81,6 +90,7 @@ catch {
     exit 1
 }
 Write-Host ('Finished downloading Windows Azure Monitor Agent')
+
```

Comment:
not: remove this extra line.

Reply:
We have extra lines after each Finish log for separating the sections